lib/kitchen/directions/bake_chapter_glossary/v1.rb

```diff
@@ -4,21 +4,85 @@ module Kitchen::Directions::BakeChapterGlossary
   class V1
     renderable
 
+    class SortableDefinition
```

Comment:
There's nothing about this class that is specific to definitions.  So I'd propose making the name of it be more generic.  What about moving this class to `lib/kitchen`:

```ruby
class I18nString < String
  def <=>(other)
    I18n.sort_strings(self, other)
  end
end
```